handlers/jiochat/jiochat.go

```diff
@@ -161,71 +161,6 @@ func buildMediaURL(mediaID string) string {
 	return mediaURL.String()
 }
 
-type fetchPayload struct {
-	GrantType    string `json:"grant_type"`
-	ClientID     string `json:"client_id"`
-	ClientSecret string `json:"client_secret"`
-}
-
-// fetchAccessToken tries to fetch a new token for our channel, setting the result in redis
-func (h *handler) fetchAccessToken(ctx context.Context, channel courier.Channel) error {
-	clog := courier.NewChannelLog(courier.ChannelLogTypeTokenRefresh, channel, h.RedactValues(channel))
-
-	tokenURL, _ := url.Parse(fmt.Sprintf("%s/%s", sendURL, "auth/token.action"))
-	payload := &fetchPayload{
-		GrantType:    "client_credentials",
-		ClientID:     channel.StringConfigForKey(configAppID, ""),
-		ClientSecret: channel.StringConfigForKey(configAppSecret, ""),
-	}
-
-	req, err := http.NewRequest(http.MethodPost, tokenURL.String(), bytes.NewReader(jsonx.MustMarshal(payload)))
-	if err != nil {
-		return err
-	}
-	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("Accept", "application/json")
-
-	resp, respBody, err := handlers.RequestHTTP(req, clog)
-	if err != nil || resp.StatusCode/100 != 2 {
-		clog.End()
-		return h.Backend().WriteChannelLog(ctx, clog)
-	}
-
-	accessToken, err := jsonparser.GetString(respBody, "access_token")
-	if err != nil {
-		clog.Error(courier.ErrorResponseValueMissing("access_token"))
-		clog.End()
-		return h.Backend().WriteChannelLog(ctx, clog)
-	}
-
-	rc := h.Backend().RedisPool().Get()
-	defer rc.Close()
-
-	cacheKey := fmt.Sprintf("jiochat_channel_access_token:%s", channel.UUID().String())
-	_, err = rc.Do("SET", cacheKey, accessToken, 7200)
```

Comment:
this has always been broken because this returns a redis syntax error.. should be `rc.Do("SET", cacheKey, accessToken, "EX", 7200)`